src/renderer/pages/Settings/Networks/Networks.tsx

```diff
@@ -42,29 +49,10 @@ export const Networks = () => {
     setTimeout(() => navigate(Paths.SETTINGS), DEFAULT_TRANSITION);
   };
 
-  const extendedChains = Object.values(chains).reduce<ExtendedChain[]>((acc, chain) => {
-    if (!includes(chain.name, query)) return acc;
-
-    const connection = connections[chain.chainId];
-    const extendedChain = {
-      ...chain,
-      connection,
-      connectionStatus: connectionStatuses[chain.chainId],
-    };
-
-    acc.push(extendedChain);
-
-    return acc;
-  }, []);
-
-  const [inactive, active] = partition(extendedChains, ({ connection }) =>
-    networkUtils.isDisabledConnection(connection),
-  );
-
   const confirmRemoveCustomNode = (name: string): Promise<boolean> => {
     return confirm({
       title: t('settings.networks.confirmModal.removeTitle'),
-      message: <Trans t={t} i18nKey="settings.networks.confirmModal.removeLabel" values={{ node: name }} />,
+      message: <Trans t={t} i18nKey="settings.networkss.confirmModal.removeLabel" values={{ node: name }} />,
```

Comment:
There's a typo here and below. I'll fix it, commenting here to remember
```suggestion
      message: <Trans t={t} i18nKey="settings.networks.confirmModal.removeLabel" values={{ node: name }} />,
```